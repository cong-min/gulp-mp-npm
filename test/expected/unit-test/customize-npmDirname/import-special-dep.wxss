@import './dist/weui-miniprogram/weui-wxss/dist/style/weui.wxss';
