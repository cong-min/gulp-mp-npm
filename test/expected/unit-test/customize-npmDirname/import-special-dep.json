{
  "usingComponents": {
    "mp-msg": "./dist/weui-miniprogram/msg/msg",
    "page": "./local-dep/page/page"
  }
}